app/views/admin/order_items/_order_item.html.erb

```diff
@@ -1,6 +1,6 @@
-<li class="order-item listing">
-  <p>Item: <%= link_to order_item.item.title, item_path(order_item.item) %></p>
-  <p>Quantity: <%= order_item.quantity %></p>
-  <p>Unit Cost: <%= in_dollars(order_item.original_unit_price) %></p>
-  <p>Subtotal: <%= in_dollars(order_item.subtotal) %></p>
+<li class="order-item listing list-group-item">
+  <h4>Item: <%= link_to order_item.item.title, item_path(order_item.item) %></h4>
```

Comment:
Not totally sure if it's conventional to use so many header tags on one page? Although seems good at this point :D